logistics_project/apps/malawi/extensions/logistics/supplypoint.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from logistics.models import Product
+from logistics.util import config
+
+
+class MalawiSupplyPointExtension(models.Model):
```

Comment:
it might be worth a comment why this is using an extension. it makes sense to me but I had to think about it for a while because it wasn't adding any additiona DB fields. I think the answer is:

1. we couldn't modify the class directly since it's supposed to be generic
2. it was easier to just swap in this way than find every usage of `commodities_stocked` and convert to a function